charts/studio/values.yaml

```diff
@@ -51,9 +51,6 @@ global:
     github:
       # -- GitHub Enterprise URL
       url: ""
-      # -- GitHub Enterprise API URL
-      apiUrl: ""
```

Comment:
We have to keep this. Otherwise it will break the GHE installation.
The bug here is that we forgot to reference this variable in `GITHUB_API_URL` in `_env_vars.tpl`. I'll submit another PR to fix this

Reply:
https://github.com/iterative/helm-charts/pull/48